alf/algorithms/ppo_algorithm.py

```diff
@@ -63,13 +58,6 @@ def preprocess_experience(self, exp: Experience):
         returns = exp.info.value + advantages
         return exp._replace(info=PPOInfo(returns, advantages))
 
-    def predict(self, time_step: ActionTimeStep, state=None):
-        return self._algorithm.train_step(time_step, state)
-
-    @property
-    def predict_state_spec(self):
-        return self._algorithm.train_state_spec
-
 
 @gin.configurable
 def create_ppo_algorithm(env, debug_summaries=False):
```

Comment:
perhaps we should add a parameter `loss_class=PPOLoss` here instead of config it in ppo.gin , it make  code easier to read